shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -471,6 +472,26 @@ TEST_F(EmbedderTest, CanSetCustomLogMessageCallback) {
   callback_latch.Wait();
 }
 
+//------------------------------------------------------------------------------
+/// Tests that setting a custom log tag works.
+TEST_F(EmbedderTest, CanSetCustomLogTag) {
```

Comment:
Maybe just a variant of this that tests that the default if "flutter"?

Reply:
The test right above it should test that already (updated the comment to that effect).